talk/owt/sdk/base/dualvideoencoder.cc

```diff
@@ -0,0 +1,33 @@
+#include "webrtc/api/video_codecs/builtin_video_encoder_factory.h"
+#include "talk/owt/sdk/base/dualvideoencoder.h"
+
+namespace owt {
+namespace base {
+
+
+DualVideoEncoder::DualVideoEncoder()
```

Comment:
currently, just a shallow wrapper over the built-in one we already use. Will add more details in a future PR.